### PR TITLE
SOC-1156 Add a backtrace to 'mediawiki' emails

### PR DESCRIPTION
**Pull request by garthwebb**
https://wikia-inc.atlassian.net/browse/SOC-1156

@WikiaSparrow

### CODE REVIEW

File: includes/wikia/WikiaMailer.php [hunk 1]
```diff
@@ -177,6 +181,37 @@ static public function send ( $headers, $to, $from, $subject, $body, $priority =
...
+		// to this function and increase the amount of context we show
+		if ( !empty( $trace[4]['function'] ) && $trace[5]['function'] != 'wfRunHooks' ) {
+			$removeCallsUpTo = 1;
+			$num += 10;
+		}
+		$trace = array_slice( $trace, $removeCallsUpTo );
+
+		$formattedTrace = [];
+		$count = $num;
+		foreach ($trace as $frame) {
```
Comment (mklucsarits) [inline includes/wikia/WikiaMailer.php]: Make sure whitespace formatting is correct inside parentheses.

File: includes/wikia/WikiaMailer.php [hunk 2]
```diff
@@ -177,6 +181,37 @@ static public function send ( $headers, $to, $from, $subject, $body, $priority =
 		return false;
 	}
 
+	static public function backtrace( $num = 8 ) {
```
Comment (mklucsarits) [inline includes/wikia/WikiaMailer.php]: Can `$num` can named more descriptively? Seems like it's a depth variable, so maybe `$depth`?

**Comment by jsutterfield** [comment]
LGTM

**Comment by mklucsarits** [comment]
LGTM

**Pull request merged**